### PR TITLE
Rlops API

### PR DESCRIPTION
**Pull request by vwxyzjn**
This PR is a massive revamp. Tagging some folks for thoughts and review @araffin @qgallouedec @simoninithomas @joaogui1 @dtch1997 @dosssman @dipamc @yooceii @Trinkle23897 @kinalmehta @pseudo-rnd-thoughts.

The main file to review is the README at https://github.com/openrlbenchmark/openrlbenchmark/tree/rlops-api

### CODE REVIEW

**Comment by vwxyzjn** [comment]
[9da2b32](https://github.com/openrlbenchmark/openrlbenchmark/pull/6/commits/9da2b32f4265d2d11b1098b915f247267aff108f) Added a caching mechanism that saves 95% of downloading time. Goes from `2:57.50 total` to `8.457 total`. In the future, we can add the cached data to huggingface's dataset, maybe?

```
python -m openrlbenchmark.rlops   67.58s user 3.81s system 40% cpu 2:57.50 total
```

```
python -m openrlbenchmark.rlops   6.60s user 2.80s system 111% cpu 8.457 total
```

**Comment by simoninithomas** [comment]
Hey there 👋 nice update 🔥 I have some small feedback on the documentation though 🤔 (but it's minor stuff) if you prefer I can open a PR but I give you the main ideas here first:

1. I think you should add an **installation element instead of getting started first with** 
a. Pip
b. Dev setup

# Installation
You can install it via pip or dev setup
## Pip install
`pip install openrlbenchmark`

## Dev setup
[section dev setup]

2. The Getting Started is really really good, **well explained to me nothing to change on this part.**

3. I would add in addition to this Getting started a colab that people **can run directly in the browser always helps to reduce friction**, but it can be tricky to do though. 

I shared the project with the HF RL team but given it's the holiday season I think you're going to get some feedback from others starting next week 🤗.

**Review by kinalmehta — APPROVED** [review]
Awesome update.  Added a comment.
Everything else LGTM. :D

File: README.md [hunk 1]
```diff
@@ -2,82 +2,281 @@
...
+    * `cen`: `exp_name` (e.g., `sac_continuous_action`, `ppo_continuous_action`, `ppo_atari`)
+    * `metric`: `charts/episodic_return`
+* [Stable-baselines3](https://wandb.ai/openrlbenchmark/sb3)
+    * `ceik`: `env`
+    * `cen`: `algo` (e.g., `sac`, `ppo`, `a2c`)
+    * `metric`: `rollout/ep_rew_mean` or `eval/mean_reward`
+* [ikostrikov/jaxrl](https://wandb.ai/openrlbenchmark/jaxrl)
+    * `ceik`: `env_name`
+    * `cen`: `algo` (e.g., `sac`)
+    * `metric`: `training/return` or `evaluation/average_returns`
```
[the remark this answers is not in the document]
Comment (vwxyzjn) [reply README.md:106]: @kinalmehta, thanks for reviewing! I agree that having a table with available metrics for all libraries could be helpful, though we should probably put it as future work.

**Comment by vwxyzjn** [comment]
@simoninithomas @kinalmehta thanks for the review. I have addressed your comments and added a colab notebook. I will merge this PR and aim for an Alpha release after the new year. Happy holiday you all :)

**Pull request merged**